.travis.yml

```diff
@@ -30,6 +29,10 @@ matrix:
       env: RAILS=master
   allow_failures:
     - env: RAILS=master
+branches:
+  only:
+    - master
+    - /\A\d-\d-stable\z/
```

Comment:
Yes, exactly. I added this change so that Travis would stop running the test suite twice for PRs made by you or me. I don't understand why you think it's a bad change.